runtime/nls/j9vm/j9vm.nls

```diff
@@ -1782,3 +1782,14 @@ J9NLS_VM_LOADING_CONSTRAINT_OVERRIDE_VIOLATION.explanation=A class loading const
 J9NLS_VM_LOADING_CONSTRAINT_OVERRIDE_VIOLATION.system_action=A java/lang/LinkageError will be thrown
 J9NLS_VM_LOADING_CONSTRAINT_OVERRIDE_VIOLATION.user_response=Examine the use of ClassLoaders to determine why a class of the same name is loaded in two different loaders
 # END NON-TRANSLATABLE
+
+# First argument is class name length
+# Second argument is class name
+# Third argument is nest host name length
+# Fourth argument is nest host name
+J9NLS_VM_NEST_HOST_FAILED_TO_LOAD=Class %2$.*1$s must be able to load its nest host %4$.*3$s.
```

Comment:
do these messages matchup with the ones OpenJDK is using?

Reply:
Some of them have different messages. The ones in which I add/directly use existing NLS messages, I will update to correspond with the ones OpenJDK is using.

What should I do for the cases in which existing error messages are different? By which I mean, the cases I did not add/change, eg. NoClassDefFound for testing the class loader case

---

I ran my updated set of test cases (currently found here: https://github.com/eclipse/openj9/pull/1201) against both. For each test that was supposed to raise an error/exception, I printed: test case name + error/exception + cause of error/exception.

Relevant test cases for this PR are:
* testDifferentClassLoadersWithAccess
* testDifferentPackagesWithAccess
* testNestMemberNotVerifiedWithAccess
* testNesthostNotClaimedWithAccess

---

Ours:
> testDifferentClassLoadersWithAccess             java.lang.NoClassDefFoundError: FieldAccess$Inner               java.lang.ClassNotFoundException

> testDifferentPackagesWithAccess         java.lang.IllegalAccessError: Class FieldAccess illegally accessing "package private" member of class pkg/FieldAccess$Inner             null

> testNestMemberNotVerifiedWithAccess             java.lang.VerifyError: Class FieldAccess$Inner must be claimed by its nest host .               null

> testNesthostNotClaimedWithAccess                java.lang.IllegalAccessError: Class FieldAccess illegally accessing "private" member of class FieldAccess$Inner         null

OpenJDK:
> testDifferentClassLoadersWithAccess             java.lang.NoClassDefFoundError: FieldAccess$Inner               java.lang.ClassNotFoundException: FieldAccess$Inner

> testDifferentPackagesWithAccess         java.lang.IllegalAccessError: failed to access class pkg.FieldAccess$Inner from class FieldAccess               null

> testNestMemberNotVerifiedWithAccess             java.lang.IllegalAccessError: Type FieldAccess$Inner is not a nest member of FieldAccess: current type is not listed as a nest member           null

> testNesthostNotClaimedWithAccess                java.lang.IllegalAccessError: tried to access field FieldAccess$Inner.f from class FieldAccess          null

---

On a side note, how close do/should the parser errors be? I've copied the same output for those cases below and they are similar but not exactly the same. 

This:
> testMultipleNestMembersAttributes               java.lang.ClassFormatError: JVMCFRE143 Multiple nest mates attributes; class=MultipleNestMembersAttributes, offset=166          null

> testMultipleNestHostAttributes          java.lang.ClassFormatError: JVMCFRE143 Multiple nest mates attributes; class=MultipleNestHostAttributes, offset=282             null

> testMultipleNestAttributes              java.lang.ClassFormatError: JVMCFRE143 Multiple nest mates attributes; class=MultipleNestAttributes, offset=188         null

OpenJDK:
> testMultipleNestMembersAttributes               java.lang.ClassFormatError: Multiple NestMembers attributes in class file MultipleNestMembersAttributes         null

> testMultipleNestHostAttributes          java.lang.ClassFormatError: Multiple NestHost attributes in class file MultipleNestHostAttributes               null

> testMultipleNestAttributes              java.lang.ClassFormatError: Conflicting NestHost and NestMembers attributes in class file MultipleNestAttributes                null